libfaad/sbr_hfadj.c

```diff
@@ -301,31 +301,24 @@ static real_t find_log2_E(sbr_info *sbr, uint8_t k, uint8_t l, uint8_t ch)
         E >>= amp1;
         if (e < 0 || e >= 64 || E < 0 || E > 24)
             return LOG2_MIN_INF;
+        E -= 12;
 
-        if (ch == 0)
+        if (ch != 0)  // L/R anti-symmetry
```

Comment:
He?

Reply:
It is a comment that left (`ch == 0`) and right (`ch != 0`) channels have joint E/Q, which means the bigger result acquires one of them, the smaller gets the other one (anti-symmetry). This change removes code repetition - easier to read and understand.